InfiniteRecharge/src/main/java/frc/robot/subsystem/navigation/NavigationSubsystem.java

```diff
@@ -73,8 +74,13 @@ public void initialize() {
         ahrs = BitBucketsAHRS.instance();
         sys = new RobotSystem();
 
+        ahrs.setAngleAdjustment(90);
```

Comment:
I assume this is because the WPI trajectory stuff thinks "forward" is facing to the right or left?

Reply:
I actually don't know, I'll investigate today what direction WPI thinks is +